src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/main.js

```diff
@@ -687,14 +702,26 @@ define([
                         this.renderPreview(data);
                     }.bind(this));
 
+                    this.sandbox.on('sulu.preview.changes', this.handleChanges.bind(this));
+
                     this.sandbox.on('sulu.preview.initialize', function(data, restart) {
+                        if (this.previewMode === 'auto') {
+                            // if mode is auto init immediately
+                            this.sandbox.emit('sulu.preview.initialize.force', data, restart);
```

Comment:
Is there a reason this is encapsulated in an own event? And why is it called force?